### ISSUE TITLE
For the electrode position, how in the code by entering xyz coordinates, there is also no position point T6

### ISSUE THREAD

**Issue by xiberJ** [simnibs/simnibs#143]
![1684753845716](https://github.com/simnibs/simnibs/assets/122353886/e1de3c7a-30f6-4567-bdef-2a2e65e4c4bc)
![1684753860607](https://github.com/simnibs/simnibs/assets/122353886/375ab52d-426a-406d-aace-aaced3f53a06)

Thanks for taking time out of your busy schedule

**Comment by xiberJ** [comment]
I've also seen a variety of positions for eeg_positions. The default is EEG10-10_UI_Jurak_2007. How do you change other positions

**Comment by oulap** [comment]
There's a few electrode layouts implemented in Simnibs, see here:

https://github.com/simnibs/simnibs/tree/master/simnibs/resources/ElectrodeCaps_MNI

Those get automatically warped to the subject scan and placed into m2m_<subid>/eeg_positions and you can change between them by changing the name you pass to the simulation script. Not sure any of the caps have T6 by default, but if you now its location in MNI space, you can just add it to the .csv files inside the resources-folder. 

In the ElectrodeCaps_MNI folder the xyz-coordinates are the electrode positions in MNI-space, whereas the ones inside eeg_positions are the subject-specific coordinates. The transformation used to warp the caps is stored in m2m<subid>/toMNI

So basically if you have the T6 coordinate in MNI space you could also use the mni2subject_coords command to map the T6 coordinate to your subject, but it's probably easier to define it in one of the caps so it happens automatically.

**Comment by xiberJ** [comment]
![1684761266747](https://github.com/simnibs/simnibs/assets/122353886/e0e9ffaa-d5bd-4d07-b78f-1fab5207a3a5)
I see that there are a variety of eeg_positions, and the default position points in EEG10-10_UI_Jurak_2007. csv are used when running through code. How do I use the other.csv positions
![1684761353649](https://github.com/simnibs/simnibs/assets/122353886/891929e3-d5be-4f77-aa21-4ae5e1d7408a)
Are the coordinates of electrode positions in mni space in the red box? If I know the coordinates of T6, I just need to format it in this folder, right?

**Comment by xiberJ** [comment]
> 在Simnibs中实践发现了一些电极布局，请参看这里：
> 
> https://github.com/simnibs/simnibs/tree/master/simnibs/resources/ElectrodeCaps_MNI
> 
> 这些会自动破解到主题扫描并放入m2m_/eeg_positions中，您可以通过更改传给模拟脚本的名称在他们之间进行更改。默认情况情况下是没有任何帽子配件T6，但如果您现在将其定位在MNI空间中，则可以将其添加到资源文件夹层的.csv文件中。
> 
> 在ElectrodeCaps_MNI文件夹中，xyz坐标是MNI空间中的电极位置，而eeg_positions中的坐标是特定于主题的坐标。用于挽歌帽的转储在m2m/toMNI中
> 
> 所以基本上，如果你在 MNI 空间中有 T6 坐标，你也可以使用 mni2subject_coords 命令将 T6 坐标映射到你的主题，但在其中一个大写中定义它可能更容易，所以它会自动发生。
![1684761266747](https://github.com/simnibs/simnibs/assets/122353886/e0e9ffaa-d5bd-4d07-b78f-1fab5207a3a5)
I see that there are a variety of eeg_positions, and the default position points in EEG10-10_UI_Jurak_2007. csv are used when running through code. How do I use the other.csv positions
![1684761353649](https://github.com/simnibs/simnibs/assets/122353886/891929e3-d5be-4f77-aa21-4ae5e1d7408a)
Are the coordinates of electrode positions in mni space in the red box? If I know the coordinates of T6, I just need to format it in this folder, right?
